src/bridge/cockpitpackages.c

```diff
@@ -386,22 +406,21 @@ read_package_manifest (const gchar *directory,
     }
   else
     {
-      g_autofree gchar *override_path = g_build_filename (directory, "override.json", NULL);
-      override = read_json_file (override_path, &error);
-      if (error)
-        {
-          if (g_error_matches (error, G_FILE_ERROR, G_FILE_ERROR_NOENT))
-            g_debug ("%s: no override found", package);
-          else
-            g_message ("%s: couldn't read override.json: %s", package, error->message);
-          g_clear_error (&error);
-        }
-      if (override)
+      /* possible override locations, in ascending priority */
+      g_autofree gchar *pkgdir_override = g_build_filename (directory, "override.json", NULL);
+      apply_override (manifest, pkgdir_override);
+
+      g_autofree gchar *package_override_name = g_strconcat (package, ".override.json", NULL);
+
+      for (const char * const *dir = cockpit_conf_get_dirs (); *dir; ++dir)
```

Comment:
This is non-idiomatic.

Something more like

```
const char * const *dirs = cockpit_conf_get_dirs ();
for (gint i = 0; dirs[i]; i++)
  ... dirs[i] ...
```
plz.

Reply:
Changed accordingly.